docs/dockercon-summit.md

```diff
@@ -1,3 +1,36 @@
-# Dockercon Summit
+# DockerCon Internals Summit
+
+## LinuxKit Summit BoF: Session 1
+
+### Incoming changes:
+  - Moby tool will move out of LinuxKit
+  - init will be rewritten in Rust
+  - pakage and kernel builds will leverage multi-stage builds
+
+### Discussion:
+- How to combine kernel flavors?  ex: Wireguard + Landlock
+  - proposal: Automated kernel patch merging for build recipes?
+  - proposal: "Security" kernel channel with combination of projects
+- Can we have a LinuxKit mailing list or use Slack?
+  - prefer mailing list
+  - need to keep community up to date with timelines for big changes such as Moby tool movement
```

Comment:
I think on this topic we also discussed creating an issue tracker/milestone with a monthly horizon so that folks can see what is planned and progress towards that.